code/modules/cargo/supplypod.dm

```diff
@@ -466,16 +466,18 @@
 		return
 	glow_effect.layer = LOW_ITEM_LAYER
 	glow_effect.fadeAway(delays[POD_OPENING])
-	remove_glow()
+	//Trust the signals
```

Comment:
wat

Reply:
comments should document what code is now not what it used to be

Reply:
I'll make it more clear, we're trusting the qdel hook signals to clean up the glow's refs